types/protocol.d.ts

```diff
@@ -2,94 +2,14 @@
  * Auto-generated by protocol-dts-generator.ts, do not edit manually. *
  **********************************************************************/
 
-export namespace Protocol {
+export module Protocol {
```

Comment:
the diff gets a little wonky depending on how far you scroll but the only changes in here are
- `s/namespace/module`
- `DomainApi` interfaces deleted
- whitespace adjustments
- descriptions for event types now included in jsdoc

Reply:
why module but not namespaces? 

Initially added the DomainApi so the types work with noice-rpc-json out of the box. I believe vscode-chrome-debugger also uses this info so this could be a breaking change for some clients.

@roblourens please confirm.

Reply:
looks like `vscode-chrome-debug-core` does indeed depend on these. We'll figure out what we should do

Reply:
> why module but not namespaces?

yeah, looks like you're right